docs/generate/readme.md

```diff
@@ -0,0 +1,388 @@
+
+### Tag: v1
+
+These settings apply only when `--tag=v1` is specified on the command line.
+
+```yaml $(tag) == 'v1'
+input-file: pets.json
+```
+
+### Tag: v2
+
+These settings apply only when `--tag=v2` is specified on the command line.
+
+```yaml $(tag) == 'v2'
+input-file: petsv2.json
+```
+
+## Python
+
+See configuration in [readme.python.md](./readme.python.md)
+
+## Java
+
+See configuration in [readme.java.md](./readme.java.md)
+````
+
+Let's now discuss what's going to be different between the two languages.
+
+1. Location of the output: We want our Python sdk to go into `azure-sdk-for-python`, and we want our Java sdk to go into `azure-sdk-for-java`. With Python, we use the flag `--python-sdks-folder` to indicate the location of our local [`azure-sdk-for-python`](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk) clone, and for Java, we indicate the location of our local [`azure-sdk-for-java`](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk) clone with the flag `--azure-libraries-for-java-folder`. This will vary based off of whether we're generating `v1` or `v2`, so we need individual conditional yaml blocks.
+2. Namespace: We want our Python namespace to be `azure.pets`, while we want our Java namespace to be `com.microsoft.azure.pets`. We want different namespaces based off of whether we're generating `v1` or `v2` as well.
+3. For Python, we also want to specify the name of our Python package with flag `package-name`
+4. Finally, for Java, we would like our library to be `fluent`
+
+Let's put all of this information into our Python readme, `readme.python.md`:
+````
+# Python
+
+These settings apply only when `--python` is specified on the command line.
+
+``` yaml
+package-name: azure-pets
+```
+
+## Tag: v1
+
+These settings apply only when `--tag=v1` is specified on the command line.
+
+```yaml $(tag) == 'v1'
+namespace: azure.pets.v1
+output-folder: $(python-sdks-folder)/pets/azure-pets/azure/pets/v1
+```
+
+## Tag: v2
+
+These settings apply only when `--tag=v2` is specified on the command line.
+
+```yaml $(tag) == 'v2'
+namespace: azure.pets.v2
+output-folder: $(python-sdks-folder)/pets/azure-pets/azure/pets/v2
+```
+````
+
+Similarly, we have our Java readme, `readme.java.md`:
+````
+# Java
+
+These settings apply only when `--java` is specified on the command line.
+
+``` yaml
+fluent: true
+```
+
+## Tag: v1
+
+These settings apply only when `--tag=v1` is specified on the command line.
+
+```yaml $(tag) == 'v1'
+namespace: com.microsoft.azure.pets.v1
+output-folder: $(azure-libraries-for-java-folder)/pets/v1
+```
+
+## Tag: v2
+
+These settings apply only when `--tag=v2` is specified on the command line.
+
+```yaml $(tag) == 'v2'
+namespace: azure.pets.v2
+output-folder: $(azure-libraries-for-java-folder)/pets/v2
+```
+````
+
+Now, when generating `v2` code in Python, our command line looks like
+```
+autorest readme.md --python --python-sdks-folder={path to local clone of azure-sdk-for-python/sdk}
```

Comment:
nit: This seems to go in the direction of azure-sdk-* and azure-rest-api-spec contributor guide and not a general-purpose autorest doc. 

Who do we expect to be the audience of this document? I wouldn't like someone in azure-sdk-for-net to follow this tutorial just to discover you don't actually have to do these things.

Reply:
that's fair. i made the docs more generic, but still included this information for python-sdks-folder etc in a note.